src/Uno.Toolkit.UI/Controls/LoadingView/LoadingView.HotReload.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+using System.Threading.Tasks;
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+#else
+using Windows.UI.Xaml;
+#endif
+using Uno.Toolkit.UI;
+
+[assembly: ElementMetadataUpdateHandlerAttribute(typeof(LoadingView), typeof(LoadingViewElementMetadataUpdateHandler))]
+[assembly: ElementMetadataUpdateHandlerAttribute(typeof(ExtendedSplashScreen), typeof(LoadingViewElementMetadataUpdateHandler))]
+
+namespace Uno.Toolkit.UI;
+internal static class LoadingViewElementMetadataUpdateHandler
+{
+
+	public static void CaptureState(FrameworkElement element, IDictionary<string, object> stateDictionary, Type[]? updatedTypes)
+	{
+		if (element is LoadingView lv)
+		{
+			if (lv.Source is { } src)
+			{
+				stateDictionary["LV.Source"] = src;
+			}
+			if (lv.Content is { } content)
+			{
+				stateDictionary["LV.Content"] = content;
+			}
+		}
+
+	}
+
+	public static Task RestoreState(FrameworkElement element, IDictionary<string, object> state, Type[]? updatedTypes)
+	{
+		if (element is LoadingView lv)
+		{
+			if (lv.Source is null && state.TryGetValue("LV.Source", out var savedSrc) && savedSrc is ILoadable src)
```

Comment:
we should probably already override the value, if a previous state has been captured
```suggestion
			if (state.TryGetValue("LV.Source", out var savedSrc) && savedSrc is ILoadable src)
```

Reply:
Actually not : if the value as been set in the ctor, it usually means that we don't want to carry the old value but keep the previous value. And actually we also need to make sure to copy the value only if it was not set using a binding, so we need to add this:


```suggestion
			if (lv.Source is null && lv.GetBindingExpression(LoadingView.SourceProperty) is null && state.TryGetValue("LV.Source", out var savedSrc) && savedSrc is ILoadable src)
```